db/migrate/20170524020316_add_details_to_games.rb

```diff
@@ -0,0 +1,8 @@
+class AddDetailsToGames < ActiveRecord::Migration[5.0]
+  def change
+    add_column :games, :number_of_moves, :integer
+    add_column :games, :white_player_id, :integer
+    add_column :games, :black_player_id, :integer
```

Comment:
For white_player_id and black_player_id, we'll want these to be indexed.

I think the syntax is `add_column :games, :black_player_id, :integer, index: true` (but double check the docs, I might be wrong). :)